### PR TITLE
Update/set microbial samples apptag in LIMS via cli, resolves issue #296

### LINKED ISSUE #296: Changing microbial apptags in LIMS
As a lab person, I want to change many microbial apptags in LIMS at the same time so that I don't need to do it manually in LIMS.

Problem: 
cg/cli/set sample only change the apptag in status-db not in LIMS.

Steps to reproduce
  1. us
  2. cg set microbial-order ORDER_ID SIGNATURE -a APPTAG

Expected outcome / suggested solution:
all samples for the have the new apptag both in statusdb and in LIMS

Questions: 
Q: 
A:

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR fixes the Changing microbial apptags in LIMS, resolves #296

How to test set apptag:
1. us
1. `cg set microbial-sample [LIMSID] [SIGNATURE] -a/--apptag [apptag]`

Expected outcome:
```
Application tag for sample [LIMSID] set to [apptag].
Comment added to sample [LIMSID]
update LIMS/application
```
microbial sample apptag has been set to the specified, an comment has been added to the order specifying who did what and when
Apptag has been set to [apptag] in LIMS for sample with [LIMSID]

### CODE REVIEW

File: cg/cli/set.py [hunk 1]
```diff
@@ -240,3 +240,7 @@ def microbial_sample(context, apptag, sample_id, user_signature):
             sample_obj.comment += '\n' + f"{timestamp}: {comment}"
         click.echo(click.style(f"Comment added to sample {sample_obj.internal_id}", fg='green'))
         context.obj['status'].commit()
+
+        print(click.style('update LIMS/application', fg='blue'))
```
Comment (ingkebil) [inline cg/cli/set.py]:
```suggestion
        click.echo(click.style('update LIMS/application', fg='blue'))
```

File: cg/cli/set.py [hunk 2]
```diff
@@ -240,3 +240,7 @@ def microbial_sample(context, apptag, sample_id, user_signature):
...
+        if context.obj.get('lims'):
```
Comment (ingkebil) [inline cg/cli/set.py]:
```suggestion
        if context.obj['lims']:
```

Comment (patrikgrenfeldt) [reply cg/cli/set.py]: It for testing without having to have lims

**Comment by patrikgrenfeldt** [comment]
@ingkebil can you approve this PR if you think I addressed you change-request?

**Comment by patrikgrenfeldt** [comment]
Thanks!

**Pull request merged**